harper-core/tests/text/linters/The Great Gatsby.snap.yml

```diff
@@ -6488,6 +6563,15 @@ Suggest:
 
 
 
+Lint:    Punctuation (31 priority)
+Message: |
+    5174 | reach Gad’s Hill until noon. Thus far there was no difficulty in accounting for
+         |                              ^~~~ Discourse markers at the beginning of a sentence should be followed by a comma.
```

Comment:
As per the previous note, this is an adverb+adjective sequence rather than a discourse marker.